test/e2e/proxy_test.go

```diff
@@ -203,6 +203,9 @@ func createPreexistingUserAndIdentity(t *testing.T, user proxyUser) (*userv1.Use
 	preexistingUser := &userv1.User{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: user.username,
+			Labels: map[string]string{
+				toolchainv1alpha1.ProviderLabelKey: toolchainv1alpha1.ProviderLabelValue,
+			},
```

Comment:
this simulates an already existing user - not created by our controller, so I would assume that it shouldn't contain the label
```suggestion
```

Reply:
There are no `User` nor `Identity` resources created in AppStudio. AppStudio tier relies on ServiceAccounts. This test case verifies that if the `User` and `Identity` resources already exist then they should not be changed nor deleted.